airbyte-integrations/connectors/source-railz/source_railz/components.py

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+import time
+from dataclasses import InitVar, dataclass
+from typing import Any, Iterable, Mapping, Optional, Union
+
+import requests
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.declarative.auth.declarative_authenticator import DeclarativeAuthenticator
+from airbyte_cdk.sources.declarative.auth.token import BasicHttpAuthenticator
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+from airbyte_cdk.sources.declarative.stream_slicers import CartesianProductStreamSlicer
+from airbyte_cdk.sources.declarative.types import Config, Record, StreamSlice
+from airbyte_cdk.sources.streams.http.requests_native_auth.abstract_token import AbstractHeaderAuthenticator
+from dataclasses_jsonschema import JsonSchemaMixin
+from isodate import Duration, parse_duration
+
+
+@dataclass
+class ShortLivedTokenAuthenticator(AbstractHeaderAuthenticator, DeclarativeAuthenticator, JsonSchemaMixin):
```

Comment:
could you please create issues / docstrings for all custom components in line with https://docs.google.com/document/d/1LB4gBk0slHj3zNmq1Qtj1U7VkKyTN9eFsVldgQ8T4HQ/edit?